ci/release/template/man/d2.1

```diff
@@ -112,6 +112,9 @@ Print debug logs.
 .It Fl -img-cache Ar true
 In watch mode, images used in icons are cached for subsequent compilations. This should be disabled if images might change
 .Ns .
+.It Fl -timeout Ar 120
+The number of seconds before d2 will timeout.
```

Comment:
"the maximum number of seconds that D2 runs for before timing out and exiting. When rendering a large diagram, it is recommended to increase this value."